php-packages/phpstan/phpstan-baseline.neon

```diff
@@ -1,17 +1,23 @@
 parameters:
 	ignoreErrors:
 		# Remove this group below with larastan 2.0 (i.e Flarum 2.0)
-		- "#Relation '[A-z_-]+' is not found in [A-z\_]+ model.#"
-		- '#^Parameter \#1 \$query of method [A-z_<>\\]+\:\:union\(\) expects [A-z_<> .,|\\]+ given\.$#'
-		- '#^Parameter \#1 \$query of method [A-z_<>\\]+\:\:joinSub\(\) expects [A-z_<> .,|\\]+ given\.$#'
+		- message: "#Relation '[A-z_-]+' is not found in [A-z\_]+ model.#"
+		  reportUnmatched: false
+		- message: '#^Parameter \#1 \$query of method [A-z_<>\\]+\:\:union\(\) expects [A-z_<> .,|\\]+ given\.$#'
+		  reportUnmatched: false
+		- message: '#^Parameter \#1 \$query of method [A-z_<>\\]+\:\:joinSub\(\) expects [A-z_<> .,|\\]+ given\.$#'
+		  reportUnmatched: false
```

Comment:
rather than ignoring each error individually, let's use the global report unmatched parameter:
https://phpstan.org/user-guide/ignoring-errors#reporting-unused-ignores

Reply:
The reason why I chose to ignore each of them instead of using the global parameter, is because I am not sure if it only affects the `ignoreErrors` of the current file of if it sets it also for user defined ones. I for one, would like to keep it enabled globally, so that my own unmatched `ignoreErrors` are still reported.

I will quickly check this.

Reply:
Indeed, by setting the parameter globally in this file, it also affects my own `ignoreErrors` in my phpstan config file. So I would prefer to keep it as so.